ui/js/controllers/jobs.js

```diff
@@ -205,6 +205,19 @@ treeherderApp.controller('ResultSetCtrl', [
             });
         };
 
+        $scope.fillAllJobs = function(revision) {
+            if (!window.confirm('This will trigger all jobs for revision ' + revision + '!\n\nAre you sure?')) {
+                return;
+            }
+
+            ThResultSetModel.fillAll($scope.resultset.id, $scope.repoName).catch(function(e) {
```

Comment:
You should give a notification to the user in case of success too. You can use .then(successCallback, failureCallback) for it.